builtInFunctions/baseAccountGuarder.go

```diff
@@ -53,32 +53,27 @@ func newBaseAccountGuarder(args BaseAccountGuarderArgs) (*baseAccountGuarder, er
 }
 
 func (baf *baseAccountGuarder) checkBaseAccountGuarderArgs(
-	senderAccount vmcommon.UserAccountHandler,
-	vmInput *vmcommon.ContractCallInput,
+	senderAddr []byte,
+	receiverAddr []byte,
```

Comment:
let's switch the position between receiverAddr and the value to prevent consecutive-same-type-of-arguments-switch :)